distro/wildfly/src/main/resources/overlay/standalone/configuration/apiman.properties

```diff
@@ -46,6 +46,9 @@ apiman.hibernate.hbm2ddl.auto=validate
 apiman-manager-ui.api.authentication.type=bearerToken
 apiman-manager-ui.api.authentication.token.generator=io.apiman.manager.ui.server.kc.KeyCloakBearerTokenGenerator
 
+# Set cross origin forn the manager; *=any, comma separated names
```

Comment:
'for' instead of 'forn'

Reply:
After this, I think it is ready for merge :-)

Reply:
ups, sorry again.